src/components/notebook/chem-results.test.tsx

```diff
@@ -0,0 +1,34 @@
+import React from "react";
+import { render, screen } from "@testing-library/react";
+import { ChemResults } from "./chem-results";
+import { ChemistryTestResult, ChemTestType } from "../../utils/chem-types";
+import { chemistryTests, chemTestRatings } from "../../utils/chem-utils";
```

Comment:
Looks like you've got some unused imports.

Reply:
Fixed.